core/agents/task_completer.py

```diff
@@ -0,0 +1,23 @@
+from core.agents.base import BaseAgent
+from core.agents.response import AgentResponse
+from core.log import get_logger
+
+log = get_logger(__name__)
+
+
+class TaskCompleter(BaseAgent):
+    agent_type = "pythagora"
+    display_name = "Pythagora"
+
+    async def run(self) -> AgentResponse:
+        self.next_state.complete_task()
```

Comment:
Suggestion: set state action string to "Task #X complete"

Reply:
👍